src/selenium/eventPage.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const EventPage = Object.create(BasePage);
+
+EventPage.getEventName = function() {
+  return this.find(By.css('h1')).getText().then(function(name) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)